docs/src/modules/java-protobuf/pages/running-locally.adoc

```diff
@@ -0,0 +1,266 @@
+----
+
+With both services configured, we can start them independently by running  [.group-java]#`mvn kalix:runAll`# [.group-scala]#`sbt runAll`# in two disinct terminals. 
+
+From a third terminal, we can create a customer on *customer-registry* service.
+
+[.tabset]
+Linux or macOS::
++
+--
+[source,command line]
+----
+grpcurl \
+ -d '{"customer_id": "vip", "email": "vip@example.com", "name": "Very Important", "address": {"street": "Road 1", "city": "The Capital"}}' \ 
+ --plaintext -d localhost:9000 \ 
+  customer.api.CustomerService/Create
+----
+--
+Windows 10+::
++
+--
+[source,command line]
+----
+grpcurl ^ 
+ -d '{"customer_id": "vip", "email": "vip@example.com", "name": "Very Important", "address": {"street": "Road 1", "city": "The Capital"}}' ^ 
+ --plaintext -d localhost:9000 ^  
+  customer.api.CustomerService/Create 
+----
+--
+
+While watching the logs in *customer-registry-subscriber* service, we will see it receiving the customer created event. After that we can query its View.
+
+[source,shell]
+----
+grpcurl --plaintext localhost:9001 customer.view.AllCustomersView/GetCustomers
+----
+
+== Running other services with Docker
+
+In the previous example, we run two services independently with [.group-java]#`mvn kalix:runAll`# [.group-scala]#`sbt runAll`#. Each time, the service is started together with a Kalix Proxy running in a Docker container.
+
+Another alternative is to build a larger `docker-compose` file containing dependent services. In the case of *customer-registry-subscriber*, we can have a second `docker-compose` file containing a Kalix Proxy for *customer-registry-subscriber*, a Kalix Proxy for *customer-registry* and the *customer-registry* itself. 
+
+At the root of the *customer-registry-subscriber* service, we can find an alternative `docker-compose` file called `docker-compose-integration.yml`. 
+
+[source,yml,indent=0]
+----
+include::java:example$java-spring-eventsourced-customer-registry-subscriber/docker-compose-integration.yml[tag=customer-registry-docker]
+----
+
+But first we need to build a image for *customer-registry*. For local development, we don't need to publish it to a remote container registry. It suffice to build it locally. 
+
+We can build the image by calling the following command at the root of  *customer-registry*:
+
+[.tabset]
+Java::
++
+--
+[source, command line]
+----
+mvn package docker:build
+----
+--
+
+Scala::
++
+--
+[source, command line]
+----
+sbt docker:publishLocal
+----
+--
+
+Next we can run *customer-registry-subscriber* and instead pass `docker-compose-integration.yml` to it. 
+
+Run the following command at the root of *customer-registry-subscriber*:
+
+[.tabset]
+Java::
++
+--
+[source, command line]
+----
+mvn -Dkalix.dev-mode.docker-compose-file=docker-compose-integration.yml kalix:runAll
+----
+--
+
+Scala::
++
+--
+[source, command line]
+----
+sbt -Dkalix.dev-mode.docker-compose-file=docker-compose-integration.yml runAll
```

Comment:
This is not yet working. The mvn plugin passed further any `-D` properties, but the sbt plugin is not yet doing it.